suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -588,6 +591,20 @@ export const getTokensFiatBalance = (
     return totalBalance.toFixed();
 };
 
+export const getStakeFiatBalance = (account: Account, coinFiatRate: Rate | undefined) => {
+    if (!coinFiatRate?.rate) return null;
```

Comment:
Can we use pass the rate itself? So that we don't have to `coinFiatRate.rate` twice in the func.